[2022-10-04 03:25:34,243] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: spark-setl3.spark_job_setl3 manual__2022-10-04T03:25:33.424593+00:00 [queued]>
[2022-10-04 03:25:34,249] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: spark-setl3.spark_job_setl3 manual__2022-10-04T03:25:33.424593+00:00 [queued]>
[2022-10-04 03:25:34,250] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 03:25:34,250] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-10-04 03:25:34,250] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 03:25:34,256] {taskinstance.py:1259} INFO - Executing <Task(SparkSubmitOperator): spark_job_setl3> on 2022-10-04 03:25:33.424593+00:00
[2022-10-04 03:25:34,261] {standard_task_runner.py:52} INFO - Started process 2714 to run task
[2022-10-04 03:25:34,263] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'spark-setl3', 'spark_job_setl3', 'manual__2022-10-04T03:25:33.424593+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/spark-setl-scala3.py', '--cfg-path', '/tmp/tmp1t_udei7', '--error-file', '/tmp/tmprvuxp0to']
[2022-10-04 03:25:34,263] {standard_task_runner.py:77} INFO - Job 50: Subtask spark_job_setl3
[2022-10-04 03:25:34,293] {logging_mixin.py:109} INFO - Running <TaskInstance: spark-setl3.spark_job_setl3 manual__2022-10-04T03:25:33.424593+00:00 [running]> on host df2389ffd8bb
[2022-10-04 03:25:34,323] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***admin@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=spark-setl3
AIRFLOW_CTX_TASK_ID=spark_job_setl3
AIRFLOW_CTX_EXECUTION_DATE=2022-10-04T03:25:33.424593+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-04T03:25:33.424593+00:00
[2022-10-04 03:25:34,329] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://10.0.9.226, Port: 7077, Schema: , Login: ***, Password: ***, extra: {}
[2022-10-04 03:25:34,330] {spark_submit.py:334} INFO - Spark-Submit cmd: spark-submit --master spark://10.0.9.226:7077 --jars /home/hoaln/work/jars/setl_2.12-1.0.0-SNAPSHOT-jar-with-dependencies.jar --class Job_s2t_AddTagVideo /home/hoaln/work/jars/setl_2.12-1.0.0-SNAPSHOT.jar
[2022-10-04 03:25:35,879] {spark_submit.py:485} INFO - 22/10/04 03:25:35 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2022-10-04 03:25:36,022] {spark_submit.py:485} INFO - 22/10/04 03:25:36 WARN DependencyUtils: Local jar /home/hoaln/work/jars/setl_2.12-1.0.0-SNAPSHOT.jar does not exist, skipping.
[2022-10-04 03:25:36,022] {spark_submit.py:485} INFO - 22/10/04 03:25:36 WARN DependencyUtils: Local jar /home/hoaln/work/jars/setl_2.12-1.0.0-SNAPSHOT-jar-with-dependencies.jar does not exist, skipping.
[2022-10-04 03:25:36,023] {spark_submit.py:485} INFO - Error: Failed to load class Job_s2t_AddTagVideo.
[2022-10-04 03:25:36,026] {spark_submit.py:485} INFO - 22/10/04 03:25:36 INFO ShutdownHookManager: Shutdown hook called
[2022-10-04 03:25:36,027] {spark_submit.py:485} INFO - 22/10/04 03:25:36 INFO ShutdownHookManager: Deleting directory /tmp/spark-1b965529-9314-4b4c-913a-9a980838e83f
[2022-10-04 03:25:36,072] {taskinstance.py:1700} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 417, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://10.0.9.226:7077 --jars /home/hoaln/work/jars/setl_2.12-1.0.0-SNAPSHOT-jar-with-dependencies.jar --class Job_s2t_AddTagVideo /home/hoaln/work/jars/setl_2.12-1.0.0-SNAPSHOT.jar. Error code is: 101.
[2022-10-04 03:25:36,077] {taskinstance.py:1277} INFO - Marking task as UP_FOR_RETRY. dag_id=spark-setl3, task_id=spark_job_setl3, execution_date=20221004T032533, start_date=20221004T032534, end_date=20221004T032536
[2022-10-04 03:25:36,086] {standard_task_runner.py:92} ERROR - Failed to execute job 50 for task spark_job_setl3
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 417, in submit
    f"Cannot execute: {self._mask_cmd(spark_submit_cmd)}. Error code is: {returncode}."
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://10.0.9.226:7077 --jars /home/hoaln/work/jars/setl_2.12-1.0.0-SNAPSHOT-jar-with-dependencies.jar --class Job_s2t_AddTagVideo /home/hoaln/work/jars/setl_2.12-1.0.0-SNAPSHOT.jar. Error code is: 101.
[2022-10-04 03:25:36,120] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-10-04 03:25:36,139] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
